tsconfig.json

```diff
@@ -23,6 +23,6 @@
       "@/*": ["./*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", ".next/types/**/*.ts"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/*.ts", ".next/types/**/*.ts"],
```

Comment:
`**/*.ts` already exists so let's undo this change.

Reply:
fixed